.stylelintrc-colors.js

```diff
@@ -0,0 +1,298 @@
+  var config = {
+    "plugins": ["stylelint-prettier"],
+    "rules": {
+      "color-no-hex": true,
+      "declaration-property-value-blacklist": [
+        {
+          "/.*/": [
+            /rgba{0,1}\(/i,
+            /hsla{0,1}\(/i,
+            /hwb\(/i,
+            /gray\(/i,
+            // uses of web color names are not allowed (with 'white' and 'black' being the only exceptions)
+            /[^\$-\w]lavender/i,
+            /^lavender/i,
```

Comment:
this is to capture the instances when a rule's value start with `<colorname>`.

e.g.,

```css
color: colorname;
// We don't have to worry about the space after `:`.
// stylelint trims out whitespaces around a rule's `value` before running the test
```